server/midddleware/namespace.go

```diff
@@ -0,0 +1,43 @@
+package middleware
+
+import (
+	"context"
+
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	api "github.com/tigrisdata/tigris/api/server/v1"
+	"github.com/tigrisdata/tigris/server/metadata"
+	"github.com/tigrisdata/tigris/server/services/v1"
```

Comment:
middleware shouldn't be dependent on the services/v1 package

Reply:
fixed.